src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py

```diff
@@ -1076,6 +1077,93 @@ def internal_get_description(dbg, seq, thread_id, frame_id, expression):
+                        arg = frame.f_locals.get('arg', None)
+                        if arg is not None:
+                            exc_type, exc_desc, trace_obj = arg
+                            break
+                    frame = frame.f_back
+
+                while trace_obj.tb_next is not None:
+                    trace_obj = trace_obj.tb_next
+
+                info = dbg.suspended_frames_manager.get_topmost_frame_and_frame_id_to_line(thread_id)
+                if info is not None:
+                    topmost_frame, frame_id_to_lineno = info
+
+                if trace_obj is not None:
+                    for frame_id, frame, method_name, filename_in_utf8, lineno in iter_visible_frames_info(
+                            dbg, trace_obj.tb_frame, frame_id_to_lineno):
+ 
+                        line_text = linecache.getline(filename_in_utf8, lineno)
+
+                        # Never filter out plugin frames!
+                        if not getattr(frame, 'IS_PLUGIN_FRAME', False):  
+                            if not dbg.in_project_scope(filename_in_utf8):
+                                if not dbg.get_use_libraries_filter():
+                                    continue
+                        frames.append((filename_in_utf8, lineno, method_name, line_text))
+            finally:
+                topmost_frame = None
+
+        if exc_desc is not None:
+            name = exc_desc.__class__.__name__
+            description = '%s' % (exc_desc,)
+        else:
+            name = 'exception: type unknown'
+            description = 'exception: no description'
+
+        stack_str = ''.join(traceback.format_list(frames))
+
+        # This is an extra bit of data used by Visual Studio 
+        source_path = frames[0][0] if frames else ''
+
+        # TODO: breakMode is set to always. This should be retrieved from exception
+        # breakpoint settings for that exception, or its parent chain. Currently json
+        # support for setExceptionBreakpoint is not implemented.
+        response = pydevd_schema.ExceptionInfoResponse(
+            request_seq=request.seq,
+            success=True,
+            command='exceptionInfo',
+            body=pydevd_schema.ExceptionInfoResponseBody(
+                exceptionId=name,
+                description=description,
+                breakMode=pydevd_schema.ExceptionBreakMode.ALWAYS,
+                details=pydevd_schema.ExceptionDetails(
+                    message=description,
+                    typeName=name,
+                    stackTrace=stack_str,
+                    source=source_path
+                )
+            )
+        )
+    except:
+        exc = get_exception_traceback_str()
+        response = pydevd_base_schema.build_response(request, kwargs={
+            'success': False,
+            'message': exc,
+            'body':{}
+        })
+    dbg.writer.add_command(NetCommand(CMD_RETURN, 0, response.to_dict(), is_json=True))
```

Comment:
Note: with the latest changes I just merged, this should pass just `response` and not `response.to_dict()`.